shuffle/cli/src/transactions.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+use crate::shared::{get_home_path, Home};
+use anyhow::{anyhow, Result};
+use diem_types::account_address::AccountAddress;
+use reqwest::{Client, Response};
+use serde_json::Value;
+use std::{cmp::max, fs, io, io::Write, str::FromStr, thread, time};
+use url::Url;
+
+// Will list the last 10 transactions and has the ability to block and stream future transactions.
+pub async fn handle(network: Url, tailing: bool) -> Result<()> {
+    let home = Home::new(get_home_path().as_path())?;
+    let address_str = fs::read_to_string(home.get_latest_address_path())?;
+    let address = AccountAddress::from_str(address_str.as_str())?;
+    let client = reqwest::Client::new();
+
+    let account_seq_num = get_account_sequence_number(&client, &network, address).await?;
+    let start = max(account_seq_num as i64 - 10, 0);
+
+    let resp = get_account_transactions_response(&client, address, &network, start, 10).await?;
+    let json_with_txns: serde_json::Value = serde_json::from_str(resp.text().await?.as_str())?;
+
+    let all_transactions = json_with_txns
+        .as_array()
+        .ok_or_else(|| anyhow::anyhow!("Failed to get transactions"))?;
+
+    for txn in all_transactions.iter() {
+        write_into(&mut io::stdout(), txn)?;
+        println!();
+        println!();
+    }
```

Comment:
You use this for...iter println! block in another place as well: L79. You could DRY this up with another method.

```rust
fn print_txns(txns) {...}
```